tensorflow/python/ops/rnn_cell_impl.py

```diff
@@ -289,6 +288,86 @@ def __call__(self, inputs, state, scope=None):
     # method.  See the class docstring for more details.
     return base_layer.Layer.__call__(self, inputs, state, scope=scope)
 
+class SRUCell(_LayerRNNCell):
```

Comment:
ick!  this *is* an API change.  this should have gone into contrib!  i thought it had been moved since the test had been moved to contrib as well!